tests/stdlib/ttypeinfo_std.nim

```diff
@@ -0,0 +1,60 @@
+import typeinfo
+
+type
+  TE = enum
+    blah, blah2
+
+  TestObj = object
+    test, asd: int
+    case test2: TE
+    of blah:
+      help: string
+    else:
+      nil
+
+
+var test = @[0,1,2,3,4]
+var x = toAny(test)
+var y = 78
+x[4] = toAny(y)
+assert x[2].getInt == 2
+
+var test2: tuple[name: string, s: int] = ("test", 56)
+var x2 = toAny(test2)
+var i = 0
+for n, a in fields(x2):
+  case i
+  of 0: assert n == "Field0" and $a.kind == "akString"
+  of 1: assert n == "Field1" and $a.kind == "akInt"
```

Comment:
I know it's pre-existing but `s/assert/doAssert`, although maybe it's better to followup with a single dedicated PR that goes through a lot of test files and does s/assert/doAssert

makes reviewing easier

Reply:
Yea, I will be careful in the future.